packages/components/src/drop-zone/style.scss

```diff
@@ -8,10 +8,12 @@
 	visibility: hidden;
 	opacity: 0;
 	border-radius: $radius-block-ui;
+	pointer-events: none;
 
 	&.is-active {
 		opacity: 1;
 		visibility: visible;
+		pointer-events: auto;
```

Comment:
Only set `pointer-events` to `auto` when active(dragging) so that users can interact with the element beneath it.